app/src/app/providers.tsx

```diff
@@ -27,12 +27,12 @@ const poppins = localFont({
     },
   ],
 });
-const openSans = localFont({
+const poppins = localFont({
```

Comment:
That makes a lot of sense, thanks for fixing the mixup :sweat_smile: I was wondering why it was always looking so different from the designs but didn't put 1 and 1 together.